api4/config.go

```diff
@@ -76,8 +76,8 @@ func configReload(c *Context, w http.ResponseWriter, r *http.Request) {
 	auditRec := c.MakeAuditRecord("configReload", audit.Fail)
 	defer c.LogAuditRec(auditRec)
 
-	if !c.App.SessionHasPermissionToAny(*c.App.Session(), model.SysconsoleReadPermissions) {
-		c.SetPermissionError(model.SysconsoleReadPermissions...)
+	if !c.App.SessionHasPermissionTo(*c.App.Session(), model.PERMISSION_RELOAD_CONFIG) {
```

Comment:
This function is only called for the `/config/reload` endpoint. It's used when the user clicks on the "Reload Configuration From Disk" button in the Web Server subsection.
I don't think read permission are sufficient to click this button because it's not a passive operation: the server's config values could change as a result of this API call.

Reply:
In that case, this change makessense.